src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import {
+  Grid,
+  Avatar as AvatarMaterialUI,
+  makeStyles,
+  CircularProgress,
+} from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import { AvatarProps } from './types';
+
+const useRegularTypeStyles = makeStyles({
+  avatar: {
+    margin: 10,
+    width: 60,
+    height: 60,
+    marginBottom: 20,
+  },
+  name: {
+    fontSize: 16,
+    fontWeight: 700,
+    lineHeight: 1.2,
+  },
+  subtitle: {
+    fontSize: 12,
+  },
+  progress: {
+    position: 'absolute',
+    top: -5,
+  },
+  grid: {
+    position: 'relative',
+  },
+});
+
+const useSmallTypeStyles = makeStyles({
+  avatar: {
+    margin: 10,
+    width: 40,
+    height: 40,
+    marginBottom: 20,
+  },
+  name: {
+    fontSize: 14,
+    fontWeight: 700,
+    lineHeight: 1.2,
+  },
+  subtitle: {
+    fontSize: 10,
+  },
+  progress: {
+    position: 'absolute',
+    top: 0,
+  },
+  grid: {
+    position: 'relative',
+  },
+});
+
+const Avatar: React.FC<AvatarProps> = props => {
+  const DEFAULT_TYPE = 'regular';
+  const DEFAULT_IMG_SRC = '/images/avatar-placeholder.jpg';
```

Comment:
it would be awesome if we could make it an `.svg` 🥇

Reply:
We can change this placeholder to some vector graphic  but I thought it's a nice touch (for now) to have raster graphics that looks like a _real photo_. Moreover users for sure will have raster images or no image at all. 

If you want I can replace Tom Hanks with sth like this (svg version)
![image](https://user-images.githubusercontent.com/14184100/67005828-d26b4a80-f0e3-11e9-8c99-322ba871ad76.png)

Reply:
Michal, I guess I wasn't clear with my comment. What I wanted to say is that we can import
the already existing material-ui icon `account_circle` for the placeholder.
Since we will need it in another screens.
https://material.io/resources/icons/?icon=account_circle&style=baseline

Reply:
Well I don't think that icons format is supported here. Material UI API supports url: `https://material-ui.com/api/avatar/`. I need url take is consumed by `<img src='path'>`

Like I've said, we can use lightweight vector here for a placeholder, like JIRA does:
[avatar-person.svg](https://docs.atlassian.com/aui/8.5.1/docs/images/avatar-person.svg)

![image](https://user-images.githubusercontent.com/14184100/67008119-2298db80-f0e9-11e9-8c12-15a8fc7e5ed8.png)

Reply:
Added Material UI icon to public assets here: 45cfd7e4692e7bca30fc6cea0891002fc1524780

![image](https://user-images.githubusercontent.com/14184100/67009123-36ddd800-f0eb-11e9-980a-3fcecdb1c530.png)

Reply:
Michal, sorry for the misunderstanding again, what I wanted you to tell is instead of importing it as `.svg` you can import the `.svg` icon already from material UI like this:

- `import AccountCircleRoundedIcon from '@material-ui/icons/AccountCircleRounded';`

and use it as:

`  <Avatar>
        <AccountCircleRoundedIcon />
      </Avatar>`

Avatar component supports children so you can don't need to use the `src` props. you can simple pass children component to it. 

So we don't need to add any placeholder manually to our folder since it's everything already in `material-ui icons`.

Let me know if there is any case that I didn't consider 👍